bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Manifest.cs

```diff
@@ -115,49 +116,49 @@ public unsafe Manifest(byte[] data, BinarySerializationEncoding encoding)
         /// <param name="contests">array of the `ContestDescription` for election</param>
         /// <param name="ballotStyles">array of the `BallotStyle` for election</param>
         public Manifest(
-            string electionScopeId, ElectionType electionType,
+            string electionScopeId, string specVersion, ElectionType electionType,
             DateTime startDate, DateTime endDate,
             GeopoliticalUnit[] gpUnits, Party[] parties,
             Candidate[] candidates, ContestDescription[] contests,
             BallotStyle[] ballotStyles)
         {
-            IntPtr[] gpUnitPointers = new IntPtr[gpUnits.Length];
+            var gpUnitPointers = new IntPtr[gpUnits.Length];

```

Comment:
i've been adjusting these according to the roslyn analyzers as i touch the files